packages/orbit-components/src/primitives/TooltipPrimitive/components/TooltipContent.js

```diff
@@ -27,50 +20,56 @@ const StyledTooltip = styled.div`
   width: 100%;
 `;
 
-const StyledTooltipWrapper = styled.div`
-  display: block;
-  position: absolute;
-  width: auto;
-  max-width: ${tooltipSize};
-  box-sizing: border-box;
-  border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
-  background-color: ${resolveBackgroundColor};
-  box-shadow: ${({ theme }) => theme.orbit.boxShadowRaised};
-  padding: ${tooltipPadding};
-  visibility: ${({ shown }) => (shown ? "visible" : "hidden")};
-  opacity: ${({ shown }) => (shown ? "1" : "0")};
-  transition: ${transition(["opacity", "visibility"], "fast", "ease-in-out")};
-  z-index: 10012; // TODO: use some good value
-  overflow-y: scroll;
-  overflow: visible;
-
-  img {
-    max-width: 100%;
-  }
-
-  // prevent position, IEs don't have initial YAY
-  top: auto;
-  bottom: auto;
-  left: auto;
-  right: auto;
-
-  // tooltip positions
-  ${resolveContainerPosition};
-  ${resolveContainerAlign};
-
-  &::after {
-    width: 0;
-    height: 0;
-    border-style: solid;
-    content: " ";
-    display: block;
-    position: absolute;
+const StyledTooltipArrow = styled(
+  React.forwardRef(({ className }, ref) => <div className={className} ref={ref} />),
+)`
+  ${({ transform, position }) => css`
```

Comment:
It would be better to prefix custom props with `$` and avoid having to forward ref manually.
```suggestion
const StyledTooltipArrow = styled.div`
  ${({ $transform, $position }) => css`
```